TaylorFramework/Modules/caprices/ProccessingMessage/CapricesExtensions.swift

```diff
@@ -49,15 +49,15 @@ extension Array {
 extension Array where Element: StringType {
     
     var containFlags: Bool {
```

Comment:
It's is used as it is, in two more files - `MessageProcessor.swift` and `MockMessageProcessor`. It would be correct to do them private&use three times in this case?